src/stories/Button/Button.tsx

```diff
@@ -9,7 +9,7 @@ interface ButtonProps {
     type?: 'button' | 'submit' | 'reset' | undefined
     selected?: boolean
     disabled?: boolean
-    onClick?: (args: any) => void | undefined
+    onClick?: (args: unknown) => void | undefined
```

Comment:
p.s. could lead to merge conflicts with https://github.com/rucio/webui/pull/65/files#diff-b86489cf2cd1640f3bbaa7c478255e6d58415bb8c8a5f4aae5bbc4c347514c79R12 

I'll rebase and merge this anyway ;)